source/overview/faq.rst

```diff
@@ -31,6 +31,79 @@ We test to 60,000 concurrent users regularly, and have a peak concurrent utiliza
 
 Mattermost provides an open source, well-documented load test simulator to verify that your Mattermost deployment can achieve the stated scale benchmarks ahead of production deployment. 
 
+What features are available on different Mattermost clients?
```

Comment:
Propose adding this question to a new "Product Questions" section, and moving below "Use Cases".